content/docs/user-guide/contributing/docs.md

```diff
@@ -51,6 +51,14 @@ Otherwise, please refer to the following procedure:
 
 We will review your PR as soon as possible. Thank you for contributing!
 
+## Automated development environment in a single click
```

Comment:
let's make it a paragraph in the Dev environment. Like a second paragraph for example should be a good fit. May remove "locally" from the first paragraph.

Reply:
Thanks! I have made the change.